login-workflow/docs/API.md

```diff
@@ -1,42 +1,40 @@
 # API
 This document outlines the various exports and configuration options for the React Auth Workflow package.
 
-The majority of the types and export used in this package come from @brightlayer-ui/react-auth-shared. You can read about those exported objects and functions in their [API](https://github.com/etn-ccis/blui-react-auth-shared/tree/master/docs/API.md) documentation.
+## RegistrationContextProviderProps
```

Comment:
This should not be here. We have created a separate doc for registration workflow guide https://github.com/etn-ccis/blui-react-workflows/pull/396/files